src/main/java/io/cryostat/rules/PeriodicArchiver.java

```diff
@@ -161,27 +134,18 @@ void performArchival() throws InterruptedException, ExecutionException {
                         .normalize()
                         .toString();
         this.logger.trace("DELETE {}", path);
```

Comment:
Likewise, this should be moved back to the Handler or removed.

Reply:
Should we keep `logger.trace("Pruning {}", recordingName);`? `performArchival()` doesn't have a similar log message

Reply:
I think that can also be removed, or moved to the PerioicArchiver.

Reply:
^ I'll remove it for consistency